src/db/queries/chunk_query.rs

```diff
@@ -83,8 +92,9 @@ pub async fn get_start_block_number_by_index(
     chunk_index: i64,
 ) -> Result<Option<i64>> {
     let stmt = format!(
-        "SELECT start_block_number FROM {} where index = $1",
-        table_name::CHUNK
+        "SELECT start_block_number FROM {}
+        where index = $1",
```

Comment:
deleted_at

Reply:
If limit `deleted_at is NULL` in this SQL, it may be NULL here 😂 (to use `start_chunk_index` to get `start_block_number`). 

And destroy the process to fetch batch, Or I should use above SQL for `start_chunk_index`.

Reply:
After discussing with @HAOYUatHZ , in commit 3b99586, fix to use chunk hash to query start and end block number, and add `deleted_at IS NULL` condition.

Tested on local with `db/tests/test.sql`.